content/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Deprecating Excalidraw Electron in favor of the Web version
+date: 2020-12-17
+author: tomayac
+link: https://twitter.com/tomayac
+---
+
+Excalidraw is a virtual collaborative whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them. On the [Excalidraw](https://excalidraw.com/) project, we have decided to deprecate [Excalidraw Desktop](https://github.com/excalidraw/excalidraw-desktop), an [Electron](https://www.electronjs.org/) wrapper for Excalidraw, in favor of the Web version that you can—and always could—find at [excalidraw.com](https://excalidraw.com/). After a careful analysis, we have decided that [Progressive Web App](https://web.dev/pwa/) (PWA) is the future we want to build upon. Read on to learn more about our rationale.
```

Comment:
> Excalidraw is a virtual collaborative whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them.

I think this intro doesn't make that much sense in the context of our blog. Maybe leave it just for the [web.dev](https://web.dev) repost?

Reply:
Absolutely, removed the first sentence.